packages/SwingSet/test/upgrade/test-upgrade.js

```diff
@@ -692,16 +649,15 @@ test('failed vatAdmin upgrade - bad replacement code', async t => {
   const kpid = c.upgradeStaticVat('vatAdmin', true, bundleID, {});
   await c.run();
   const vaUpgradeStatus = c.kpStatus(kpid);
-  const vaUpgradeCapdata = c.kpResolution(kpid);
+  const vaUpgradeResult = kunser(kunser(c.kpResolution(kpid)));
```

Comment:
Because the payload of a resolve, which is what the inner unserialization extracts, is itself a capdata.  We always had this double unserialization; the improved coding idioms just make it obvious.